integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1746,13 +1746,20 @@ func (s *DockerNetworkSuite) TestConntrackFlowsLeak(c *testing.T) {
 	cli.DockerCmd(c, "network", "create", "--subnet=192.168.10.0/24", "--gateway=192.168.10.1", "-o", "com.docker.network.bridge.host_binding_ipv4=192.168.10.1", "testbind")
 	assertNwIsAvailable(c, "testbind")
 
+	// Replaces the image built from https://github.com/appropriate/docker-nc/tree/master/latest
+	imgName := "moby/nc"
+	icmd.RunCmd(icmd.Cmd{
+		Command: []string{dockerBinary, "build", "-t", imgName, "-"},
+		Stdin:   strings.NewReader("FROM alpine:3.14\nRUN apk add --no-cache netcat-openbsd\n"),
+	}).Assert(c, icmd.Success)
+
 	// Launch the server, this will remain listening on an exposed port and reply to any request in a ping/pong fashion
 	cmd := "while true; do echo hello | nc -w 1 -lu 8080; done"
-	cli.DockerCmd(c, "run", "-d", "--name", "server", "--net", "testbind", "-p", "8080:8080/udp", "appropriate/nc", "sh", "-c", cmd)
+	cli.DockerCmd(c, "run", "-d", "--name", "server", "--net", "testbind", "-p", "8080:8080/udp", imgName, "sh", "-c", cmd)
 
 	// Launch a container client, here the objective is to create a flow that is natted in order to expose the bug
 	cmd = "echo world | nc -q 1 -u 192.168.10.1 8080"
```

Comment:
```suggestion
	cmd = "echo world | nc -w 1 -u 192.168.10.1 8080"
```

Reply:
(I ran these exact commands in two containers and both worked "correctly" although the server container having a constant repeated 1 second timeout is kind of cute)